### ISSUE TITLE
'husky' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by ChrisMBarr** [martijnversluis/ChordSheetJS#1153]
Now that #907 has been fixed (yay!) I would like to use ChordSheetJS in my own project, but when I run `npm install chordsheetjs` I get the following errors

```
npm ERR! code 1
npm ERR! path C:\<path-to-my-project>\node_modules\chordsheetjs
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c husky install
npm ERR! 'husky' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```

From what I can gather `husky` is some kind of pre-commit git hook thing used when developing, but why is it needed as a dependency just to run or install this? I am not trying to make a PR here, I am just trying to use this package.

I suppose I could just figure out how in install `husky` on my machine even id I don't need it, but then I get the feeling this would be an issue when github tries to do CI with my project.

**Comment by martijnversluis** [comment]
Hey @ChrisMBarr. Thanks for reaching out.

When installing through NPM, Husky should not be required indeed. I'm gonna investigate this issue!

**Comment by martijnversluis** [comment]
@ChrisMBarr I just published `9.0.2.`, that version should work correctly.

**Comment by ChrisMBarr** [comment]
Excellent, I'm glad that was a simple fix. I've just installed it successfully now. Thanks!
